kairos-server/src/main.rs

```diff
@@ -1,21 +1,13 @@
-use std::net::SocketAddr;
-
 use dotenvy::dotenv;
-use kairos_server::{config::ServerConfig, state::BatchStateManager};
+use kairos_server::{config::ServerConfig, run};
 
 #[tokio::main]
 async fn main() {
-    tracing_subscriber::fmt::init();
-    dotenv().ok();
-
+    // loads the environment from the current directories .env file
+    // if the .env does not exist in the current directory,
+    // we still go ahead and try to obtain a server config from the environment
+    let _ = dotenv();
     let config = ServerConfig::from_env()
         .unwrap_or_else(|e| panic!("Failed to parse server config from environment: {}", e));
-
-    let app = kairos_server::app_router(BatchStateManager::new_empty());
-
-    let axum_addr = SocketAddr::from(([127, 0, 0, 1], config.port));
-
-    tracing::info!("starting http server on `{}`", axum_addr);
-    let listener = tokio::net::TcpListener::bind(axum_addr).await.unwrap();
-    axum::serve(listener, app).await.unwrap();
+    run(config).await
```

Comment:
NIT: I'd prefer if we used the qualified path `kairos_server::run` for clarity.

Reply:
done in [204ca19](https://github.com/cspr-rad/kairos/pull/89/commits/204ca19a4fc22b9bd8f69d8e9228ae58025663d1)